caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -661,6 +679,25 @@ def add_query(q):
             input_token_count=input_token_count,
         )
 
+    @TokenizationTask.taskmethod()
+    def run_tokenizer(
+        self,
+        text: str,
+    ) -> TokenizationResults:
+        """Run tokenization task against the embeddings model.
+
+        Args:
+           text: str
+                Text to tokenize
+        Returns:
+            TokenizationResults
+                The token count
+        """
+        if self.model:
```

Comment:
does every embeddings model supports tokenization via this method?

Reply:
I think every sentenceTransformer, yes -- as per the [docs in here](https://www.sbert.net/docs/package_reference/SentenceTransformer.html#sentence_transformers.SentenceTransformer.tokenize) cc: @markstur

Reply:
@gkumbhat @evaline-ju issue with linting is solved!! May we move forward with this one?

Reply:
All checks have passed!